pytest_order/sorter.py

```diff
@@ -138,8 +144,11 @@ def handle_dependency_mark(
             name_mark = item.node_id
         aliases[name_mark] = item
 
-    def handle_order_mark(self, item: Item) -> Optional[int]:
+    def handle_order_mark(self, item: Item) -> None:
         mark = item.item.get_closest_marker("order")
+        if mark is None:
+            # failsafe
```

Comment:
Could this also done by a cast instead of the check?

Reply:
Will do!